src/main/java/core/basesyntax/Random.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public class Random {
+    private static java.util.Random random = new java.util.Random();
```

Comment:
As much as I like the idea, you're not supposed to use static methods in this exercise: https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#dont-use-static-methods-in-your-solution

Proper util class would require you to implement the singleton pattern, which might be a bit challenging now, so let's stick with non-static objects.
Besides that, there's already a static way to get random numbers: `java.lang.Math.random()`

Please remove this class.